src/Illuminate/Mail/Mailables/Content.php

```diff
@@ -52,16 +59,18 @@ class Content
      * @param  string|null  $html
      * @param  string|null  $text
      * @param  string|null  $markdown
+     * @param  string|null  $raw
      * @param  array  $with
      *
      * @named-arguments-supported
      */
-    public function __construct(string $view = null, string $html = null, string $text = null, $markdown = null, array $with = [])
+    public function __construct(string $view = null, string $html = null, string $text = null, string $markdown = null, string $raw = null, array $with = [])
```

Comment:
This is a major breaking change. You can't add a new argument here in the middle.